AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -1166,6 +1167,9 @@ leiderdorpsweekblad.nl#%#Object.defineProperty(window,'canRunAds',{value:true});
 @@||pln-pskov.ru/js/ads.js
 exabytetv.info###deadblocker_dialog
 exabytetv.info##.deadblocker-header-bar
+||evoload.io/ads.js
+||streameast.live/wp-content/plugins/yalaan/js/deblocker.min.js
+||desktophut.com/abDetector.min.js
```

Comment:
Ok. 
Also blocked ad leftover in player
https://github.com/AdguardTeam/AdguardFilters/commit/59ad551cf8f4b41556a83b6f6935fca483212aa9